src/core_plugins/kbn_vislib_vis_types/public/controls/point_series/series.html

```diff
@@ -24,9 +26,12 @@
           ng-class="{ 'fa-caret-down': isSeriesOpen, 'fa-caret-right': !isSeriesOpen }"
           class="kuiIcon fa-caret-right kuiSideBarCollapsibleTitle__caret"
         ></span>
-        <span class="kuiSideBarCollapsibleTitle__text">
-          {{chart.data.label}}
-        </span>
+        <span
+          class="kuiSideBarCollapsibleTitle__text"
+          i18n-id="kbnVislibVisTypes.controls.pointSeries.series.chartDataLabel"
+          i18n-default-message="{chartDataLabel}"
```

Comment:
We do not need to translate places where there are variable but there are no text